HousingRepairsOnlineApi/Controllers/RepairController.cs

```diff
@@ -23,7 +31,17 @@ public async Task<IActionResult> SaveRepair([FromBody] RepairRequest repairReque
             try
             {
                 var result = await saveRepairRequestUseCase.Execute(repairRequest);
-
+                switch (repairRequest?.ContactDetails?.Type)
```

Comment:
The logic which communication method to use shouldn't _necessarily_ reside in the controller.

Creation of another type, i.e. `IAppointmentConfirmationSender`, to encapsulate the logic of deciding how to send a notification and that type injected to the controller would be recommended.